tools/rna_tools/rcas/rcas.xml

```diff
@@ -0,0 +1,306 @@
+<tool id="rcas" name="RNA Centric Annotation System" version="1.1.1">
+    <requirements>
+            <requirement type="package" version="1.1.1">bioconductor-rcas</requirement>
+    </requirements>
+    <stdio>
+        <regex match="Execution halted"
+           source="both"
+           level="fatal"
+           description="Execution halted." />
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+            Rscript $script_file
```

Comment:
The configfile is stored by Galaxy in some temp path. And this can container whitespaces or whatever. So yes :)